### PR TITLE
Remove `jsxRuntime` pragma directives

### PR DESCRIPTION
**Pull request by amoore108**
Related to https://github.com/bbc/simorgh/pull/10809

Overall changes
======
- I made a change awhile ago to automatically use the `classic` JSX Runtime so that we didn't need to include the pragma directive `/** @jsxRuntime classic */` on every file used by the Next.js app. This appears to have worked, but the cleanup of the other references was never done, so here it is!

Code changes
======
- Remove all references to `/** @jsxRuntime classic */` as this should now be handled by Babel

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Comment by HarveyPeachey** [comment]
Using your PR as guinea pig for chromatic, looks like theres still some flakiness, although most of them have gone

**Comment by amoore108** [comment]
> Using your PR as guinea pig for chromatic, looks like theres still some flakiness, although most of them have gone

No worries, work away!

File: ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.tsx [hunk 1]
```diff
@@ -1,6 +1,4 @@
 /** @jsx jsx */
-/** @jsxRuntime classic */
-/* @jsxFrag React.Fragment */
```
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.tsx:3]: Did we mean to delete this?

**Review by karinathomasbbc — APPROVED** [review]
Nice, thanks for sorting this, means we shouldn't have to worry about it in future!

File: ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.tsx (hunk 1 above)
Comment (amoore108) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.tsx:3]: Yea its unneeded in this file as we aren't using any Fragment or `<></>` elements.

Comment (karinathomasbbc) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.tsx:3]: Cool! Just wanted to check in case it was accidentally deleted 👍

**Review by Isabella-Mitchell — APPROVED** [review]

**Pull request merged**